onnxruntime/core/session/inference_session.cc

```diff
@@ -220,11 +222,73 @@ common::Status InferenceSession::Load(std::function<common::Status(std::shared_p
   return status;
 }
 
+std::shared_ptr<SchemaRegistryManager> InferenceSession::CreateSchemaRegistryManager()
```

Comment:
Randy, Ryan and I discussed this. The idea of adding a schema on the fly while loading the graph doesn't seem quite right. Schema should be supplied as part of Model::Load. More discussion to follow.